### DISCUSSION TITLE
ColumnResizeMode Struggles

### DISCUSSION THREAD

**Discussion by mdpnixon** [primefaces/community#259] [PrimeFaces]
I don't know if this is possible but I am trying to have my table resize functions so that if we reduce the total widths to be less than the table width it stays at the set width and just adds the difference to last column. And then if we expand to wider than the table width we stay at the same size and just add an overflow-x scroll to the table.

I have been able to achieve the overflow part of this using columnResizeMode="expand", but when we reduce to less than the table width the table shrinks to less than the original width. Is there a way for me to achieve this?

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.
